documentation/faq.asciidoc

```diff
@@ -91,18 +91,26 @@ That means each event needs to be completely self-describing: an event's key and
 
 Meanwhile, the Kafka Connect services serialize the connector's events and record them in Kafka. The JSON converter is very generic and very simple, but it has no choice but to serialize the entire event information. Therefore, events represented in JSON are indeed verbose and large.
 
-However, http://docs.confluent.io/{confluent-platform-version}/schema-registry/docs/index.html[Confluent's Avro Converter] is much smarter in two ways. First, it converts the connector's schema into an https://avro.apache.org[Apache Avro schema], so the payload can be serialized into a https://martin.kleppmann.com/2012/12/05/schema-evolution-in-avro-protocol-buffers-thrift.html[very compact binary form]. Secondly, it uses the fact that many events in a row will use the same schema (and thus Avro schema), and by registering those Avro schemas in a separate http://docs.confluent.io/{confluent-platform-version}/schema-registry/docs/index.html[Schema Registry], it can place into each serialized event a small _identifier_ of the schema version used by the message. The Avro Converter and the Schema Registry can work together to track the history of each schema over time.
+However, https://github.com/Apicurio/apicurio-registry[Apicurio Registry converters] are much smarter.
```

Comment:
I'd present it slightly differently, at first addressing the advantages of a schema registry altogether. In particular as the registry advantage now is there no matter whether JSON or Avro is used. I.e. there's two steps to more concise messages:

* put the message schema into a registry
* use Avro, making the payload itself smaller

Then, the two options for a registry could be described.

Reply:
@gunnarmorling Updated